sources/web/datalab/server.ts

```diff
@@ -153,6 +153,12 @@ function handleRequest(request: http.ServerRequest,
     return;
   }
 
+  if (requestPath.indexOf('/api/basepath') === 0) {
```

Comment:
Just to make sure I'm reading this correctly, nothing is yet actually making a request for /api/basepath, right?